controllers/app.controller.js

```diff
@@ -21,12 +21,15 @@ exports.getArticles = (req, res, next) => {
 }
 
 exports.getArticleById = (req, res, next) => {
-    const articleId = parseInt(req.params.article_id)
-    
-    fetchArticleById(articleId).then((article) => {
-       res.status(200).send({article})
-    })
-    .catch((err) => {
-        next(err)
-    })
+    const articleId = req.params.article_id
+    if(!parseInt(articleId)) {
```

Comment:
I like your thinking but this check won't work if the string passed starts with a number, even if the rest of the articleID is invalid. 

Think about how you can handle psql errors

Reply:
thanks carrie! Made these changes with Kevs help and he was happy for me to commit without the PR. Really sorry about my terrible branching as well - I've had a talking to!